.env

```diff
@@ -1,16 +1,17 @@
-ISV_PAYMENT_MERCHANT_ID = 
-ISV_PAYMENT_MERCHANT_KEY_ID = 
-ISV_PAYMENT_MERCHANT_SECRET_KEY = 
+ISV_PAYMENT_MERCHANT_ID = wiproltd
```

Comment:
we need to find a way to separate these values for your development versus the true file people will use to edit themselves. Essentially just ignore the future changes to this file unless we need to add a field.